gprofiler/merge.py

```diff
@@ -250,9 +252,11 @@ def concatenate_profiles(
 
     for pid, stacks in process_profiles.items():
         container_name = _get_container_name(pid, docker_client, add_container_names)
+        application_name = get_application_name(pid) or "" if add_application_names else ""
```

Comment:
Use consistent naming for the feature across the codebase. Here you have `add_application_names`. Previously I saw `separate_applications`. I'd go with the former.

Reply:
I agree with the name suggestion of @Jongy. As I wrote [here](https://github.com/Granulate/gprofiler/pull/242#discussion_r763772060) applications "separation" isn't an indicative terminology. The module finds the name of applications, so something like `AppNameGetter` or `AppNameIdentifier` will more suitable.

Reply:
Changed to `ApplicationIdentifier` I think it's suits good